src/libs/Accessibility/index.js

```diff
@@ -0,0 +1,44 @@
+import {useEffect, useState, useCallback} from 'react';
+import {AccessibilityInfo} from 'react-native';
+import _ from 'underscore';
+import moveAccessibilityFocus from './moveAccessibilityFocus';
+
+const useScreenReaderStatus = () => {
+    const [isScreenReaderEnabled, setIsScreenReaderEnabled] = useState(false);
+    useEffect(() => {
+        const subscription = AccessibilityInfo.addEventListener('screenReaderChanged', setIsScreenReaderEnabled);
+
+        return subscription.remove;
+    }, []);
+
+    return isScreenReaderEnabled;
+};
+
+const getHitSlopForSize = ({x, y}) => {
+    const minimumSize = 44;
+    const hitSlopVertical = _.max([minimumSize - x, 0]);
+    const hitSlopHorizontal = _.max([minimumSize - y, 0]);
+    return {
+        top: hitSlopVertical,
+        bottom: hitSlopVertical,
```

Comment:
Should this be `-hitSlopVertical`? (and should `right` be `-hitSlopHorizontal`?

Not sure I quite get the coordinates here, so it might be irrelevant but figured I'd call it out in case.

Reply:
These values are added to the original area's size to expand it so all should be positive values

Reply:
Yeah @dangrous the `hitSlop` I don't think negative numbers make sense for `hitSlop`. AFAIK, it's not like a coordinate system but just scalar values that extend the Pressable's range beyond its bounds (but never beyond the bounds of the parent or into the bounds of a sibling)

Reply:
oh okay that makes sense. I was thinking more coordinates